pkgs/top-level/all-packages.nix

```
@@ -28992,6 +28992,23 @@ with pkgs;
 
   spacx-gtk-theme = callPackage ../data/themes/gtk-theme-framework { theme = "spacx"; };
 
+  inherit
+    ({
+      gruppled-black-cursors = callPackage ../data/icons/gruppled-cursors { theme = "gruppled_black"; };
+      gruppled-black-lite-cursors = callPackage ../data/icons/gruppled-lite-cursors {
+        theme = "gruppled_black_lite";
+      };
+      gruppled-white-cursors = callPackage ../data/icons/gruppled-cursors { theme = "gruppled_white"; };
+      gruppled-white-lite-cursors = callPackage ../data/icons/gruppled-lite-cursors {
+        theme = "gruppled_white_lite";
+      };
+    })
+    gruppled-black-cursors
+    gruppled-black-lite-cursors
+    gruppled-white-cursors
+    gruppled-white-lite-cursors
+    ;
```

```suggestion
    gruppled-black-cursors = callPackage ../data/icons/gruppled-cursors {
      theme = "gruppled_black";
    };
    gruppled-black-lite-cursors = callPackage ../data/icons/gruppled-lite-cursors {
      theme = "gruppled_black_lite";
    };
    gruppled-white-cursors = callPackage ../data/icons/gruppled-cursors {
      theme = "gruppled_white";
    };
    gruppled-white-lite-cursors = callPackage ../data/icons/gruppled-lite-cursors {
      theme = "gruppled_white_lite";
    };
```
Isn't this just a more complicated form of this?

---

This is what I used initially, but the by-name ci was not happy with it.
Therefore I asked on https://discourse.nixos.org/t/rfc140-what-about-package-with-multiple-names/41846

---

I don't like it either...